mocks/mock-data/disputes.ts

```diff
@@ -7,6 +7,200 @@ const getDate = (daysOffset = 0, originDate = new Date()) => {
+    },
+};
+
+export const CHARGEBACK_ACCEPTED: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'ACCEPTED',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+};
+
+export const CHARGEBACK_PENDING_DEFENDABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'UNDEFENDED',
+        defensibility: 'DEFENDABLE',
+    },
+};
+export const CHARGEBACK_PENDING_DEFENDABLE_EXTERNALLY: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'UNDEFENDED',
+        defensibility: 'DEFENDABLE_EXTERNALLY',
+    },
+};
+export const CHARGEBACK_PENDING_ACCEPTABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'UNDEFENDED',
+        defensibility: 'ACCEPTABLE',
+    },
+};
+
+export const CHARGEBACK_PENDING: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    defense: undefined,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'PENDING',
+        defensibility: 'DEFENDABLE',
+    },
+};
+
+export const CHARGEBACK_PENDING_NOT_ACTIONABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...CHARGEBACK_PENDING.dispute,
+        defensibility: 'NOT_ACTIONABLE',
+    },
+};
+export const CHARGEBACK_DEFENDED: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'RESPONDED',
+    },
+    defense: {
+        ...DEFAULT_DETAIL_DEFENSE,
+        suppliedDocuments: ['Document1', 'Document2'],
+    },
+};
+export const CHARGEBACK_DEFENDED_EXTERNALLY: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'RESPONDED',
+    },
+    defense: {
+        ...DEFAULT_DETAIL_DEFENSE,
+        defendedThroughComponent: false,
+    },
+};
+
+// REQUEST FOR INFORMATION
+
+export const RFI_UNRESPONDED_DEFENDABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'UNRESPONDED',
+        type: 'REQUEST_FOR_INFORMATION',
+        defensibility: 'DEFENDABLE',
+    },
+};
+
+export const RFI_UNRESPONDED_ACCEPTABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'UNRESPONDED',
+        type: 'REQUEST_FOR_INFORMATION',
+        defensibility: 'ACCEPTABLE',
+    },
+};
+
+export const RFI_PENDING: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'PENDING',
+        type: 'REQUEST_FOR_INFORMATION',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+};
+
+export const RFI_EXPIRED: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
```

Comment:
Can use `DEFAUL_DETAIL_DISPUTE` here instead.
```suggestion
        ...DEFAULT_DETAIL_DISPUTE,
```

Reply:
✅